pyi.py

```diff
@@ -115,6 +125,40 @@ def run_check(self, plugin, **kwargs):
 
 
 @attr.s
+class PyiVisitor(ast.NodeVisitor):
+    filename = attr.ib(default='(none)')
+    errors = attr.ib(default=attr.Factory(list))
+
+    def visit_Assign(self, node: ast.Assign) -> None:
+        """Attempt to find assignments to type helpers (typevars and aliases), which should be
+        private.
+        """
+        self.generic_visit(node)
+        if isinstance(node.value, ast.Call) and isinstance(node.value.func, ast.Name) and \
+                node.value.func.id == 'TypeVar':
+            for target in node.targets:
+                if not isinstance(target, ast.Name):
+                    self.error(target, Y001)
```

Comment:
Just wondering, is anybody really doing this? I cannot find an example in typeshed.